lib/commons/aria/lookup-table.js

```diff
@@ -1637,7 +1637,9 @@ lookupTable.role = {
         'aria-level',
         'aria-selected',
         'aria-rowindex',
-        'aria-errormessage'
+        'aria-errormessage',
+        'aria-setsize',
+        'aria-posinset'
```

Comment:
The lookup table has been deprecated in favor of using `lib/standards`. We don't need to update it anymore.  It'll just be removed in 5.0.

```suggestion
        'aria-errormessage'
```